src/components/Data/Collections/CollectionList.vue

```diff
@@ -365,23 +365,23 @@ export default {
       )
     },
     async onDeleteModalSuccess() {
-      await this.fetchCollections()
+      await this.fetchCollectionsList()
```

Comment:
```suggestion
      await this.fetchCollectionList()
```

In English, when you have a list of items you say "Item List", not "Items List" (the object of the list is always singular, since the List expresses the plurality).